recipes/accimage/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "accimage" %}
+{% set version = "0.1" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/pytorch/accimage.git
+  git_rev: '1e81f95'
```

Comment:
Please do not use repos to get sources: https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

Reply:
I would have used a tarball if one is provided, however sadly there aren't any formal releases of accimage. Is this a deal breaker, or is it acceptable? I can try and get the release tagged upstream

Reply:
This is pretty much a deal breaker if the project doesn't even have a version tagged.  GitHub will generate a tarball for each tag, so if accimage had a tag you could use that.  The reason that this is problematic is that we can't be sure that the project won't release a v0.1 in the future, which will then conflict with this package.

Reply:
Fair enough. I'll try and get upstream to tag it and report back once that's happened.

Reply:
Thanks!

Reply:
OK, tagged and switched to using the github release tarball :+1: